modules/private-clusters-default.adoc

```diff
@@ -25,10 +29,28 @@ endif::aws-gov[]
 
 By default, {product-title} is provisioned to use publicly-accessible DNS and endpoints. A private cluster sets the DNS, Ingress Controller, and API server to private when you deploy your cluster. This means that the cluster resources are only accessible from your internal network and are not visible to the internet.
 
-To deploy a private cluster, you must use existing networking that meets your requirements. Your cluster resources might be shared between other clusters on the network.
+To deploy a private cluster, you must:
+
+* Use existing networking that meets your requirements. Your cluster resources might be shared between other clusters on the network.
+* Deploy from a machine that has access to:
+** The API services for the cloud to which you provision.
+** The hosts on the network that you provision.
+** The internet to obtain installation media.
+
+You can use any machine that meets these access requirements and follows your company's guidelines. For example, this machine can be a bastion host on your cloud network or a machine that has access to the network through a VPN.
```

Comment:
Since VPN is not supported in AWS China, how about delete `or a machine that has access to the network through a VPN`?